apps/wallet/src/ui/app/shared/header/Header.tsx

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { type ReactNode } from 'react';
+import { Link } from 'react-router-dom';
+
+import Logo from '../../components/logo';
+import { type API_ENV } from '_src/shared/api-env';
+
+type HeaderProps = {
+    networkName: API_ENV;
+    middleContent?: ReactNode;
+    rightContent?: ReactNode;
+};
+
+/**
+ * General page header that can render arbitrary content where the content
+ * located in the middle of the header is centered and has a capped width
+ */
+export function Header({
+    networkName,
+    middleContent,
+    rightContent,
+}: HeaderProps) {
+    return (
+        <header className="grid grid-cols-header items-center gap-3 px-3 py-1">
+            <div>
+                <Link to="/tokens" className="no-underline text-gray-90">
```

Comment:
I feel like `/` is a better place for this to go to, even though that redirects to `/tokens`, just so that we can change the home page without needing to touch multiple places